vscode-extensions/setup.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 echo 'debconf debconf/frontend select Noninteractive' | sudo debconf-set-selections
-cd /tmp && apt update && apt install bsdtar -y
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - # add the latest key
```

Comment:
Just so I can follow, what aspect of this requires installing the Yarn public key? I don't see that as a problem. I am just curious which package or extension makes that necessary.

Reply:
Oh! VSCode uses yarn internally?! TIL

Reply:
Adding the yarn public key was to fix the error when installing `bsdtar`
```
The following signatures were invalid: EXPKEYSIG 23E7166788B63E1E Yarn Packaging <yarn@dan.cx>
```

But this seems no longer needed installing `libarchive-tools` instead of `bsdtar`. I've removed it in https://github.com/katacoda/scenario-examples/pull/52/commits/181487d3642945a5d70984f099846936e807f4b2